class Error(Exception):
    pass


class LoginFailedError(Error):
    pass
